globus_automate_client/action_client.py

```diff
@@ -1,15 +1,16 @@
 import uuid
-from typing import Any, Iterable, Mapping, Optional
+from typing import Any, Iterable, Mapping, Optional, Type, TypeVar, Union
 
 from globus_sdk import (
     AccessTokenAuthorizer,
     ClientCredentialsAuthorizer,
     GlobusHTTPResponse,
     RefreshTokenAuthorizer,
 )
-from globus_sdk.authorizers import GlobusAuthorizer
 from globus_sdk.base import BaseClient
 
+_ActionClient = TypeVar("_ActionClient", bound="ActionClient")
```

Comment:
Another usage I hadn't seen/used before. I think this better than having the string elsewhere.

Gotta love python, many ways to do everything...